### DISCUSSION TITLE
Welcome to Ezra Bible App Discussions!

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#244] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Ezra Bible App community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by talltrees99** [comment]
I just installed Ezra on my Linux Mint.  I'm used to the Olive Tree Bible app on Windows, so this is a change.   I'm trying to figure out how to change the default "paste" option. so when I copy a verse from Ezra into the clipboard, it will paste into LibreOffice with the scripture reference FIRST, before the body of the verse (same line), instead of trailing at the end.  Is it possible to do that?

**Reply by tobias-klein** [reply]
@talltrees99 Thanks for engaging in this discussion! The order of verse reference and verse content in copy/pasted texts is currently fixed. If such a feature is relevant to several users, we could consider making this configurable.

**Comment by talltrees99** [comment]
Thanks for your response! In the Olive Tree app I'm used to using, that is configurable. I can paste the verses easily into my Bible study notes, with the reference first, followed immediately by the verse. I find that is very useful, and would definitely appreciate the option.

Kudos for all the effort that you've put into this app to make it so useful already!

Sent from Proton Mail Android

-------- Original Message --------
On 3/9/25 11:17 AM, Tobias Klein wrote:
@talltrees99 Thanks for engaging in this discussion! The order of verse reference and verse content in copy/pasted texts is currently fixed. If such a feature is relevant to several users, we could consider making this configurable.

**Reply by tobias-klein** [reply]
I created issue #1218 to track this enhancement request.

**Comment by cscurry1982-bit** [comment]
Just switched over to Linux and was excited to see a native Bible app that was also in the iOS app store so thank you for that. Really like how the word study pane shows you all the information available about a particular word from every resource you have installed. 

Two questions:
1) Is there is a feature in the commentary pane that will show you the commentary for an entire chapter of the Bible being viewed as opposed to just the specific verse highlighted?
2) Is there support for viewing images if they exist in a resource (commentary or dictionary)?

**Reply by tobias-klein** [reply]
Hi @cscurry1982-bit!
I'm glad you discovered this app for your Bible study!
1) Commentaries can only be viewed on a verse by verse basis at the moment.
2) Images in commentary or dictionary sword modules should be rendered. This was implemented recently.
